### PR TITLE
Fix for build failure on Flutter 3.11.0-16.0.pre

### PR DESCRIPTION
**Pull request by giuliatesta**
Added handling of AppLifecycleState.hidden case according to
https://docs.flutter.dev/release/breaking-changes/add-applifecyclestate-hidden#migration-guide

### CODE REVIEW

File: lib/src/ui/reader_widget.dart [hunk 1]
```diff
@@ -192,12 +192,14 @@ class _ReaderWidgetState extends State<ReaderWidget>
...
       case AppLifecycleState.inactive:
       case AppLifecycleState.paused:
+      case AppLifecycleState.hidden:
         controller?.dispose();
         setState(() {
           _isCameraOn = false;
         });
         break;
       case AppLifecycleState.detached:
+      default:
```
Comment (softkot) [inline lib/src/ui/reader_widget.dart:202]: That's not a good idea.

**Comment by khoren93** [comment]
@giuliatesta Thanks!

**Comment by Chappie74** [comment]
This might need to be reverted or handled differently.  As @softkot  mentioned this wasn't a good idea. 

The current version of the package `1.4.0` which is catered to target flutter ` 3.7.0`  minimum breaks all version prior to Flutter 3.11.0-16.0.pre


None of the below version currently build on any flutter <3.11.0-16.0.pre. 
image

this exception is thrown as those version don't yet have that `hidden` value in the enum. 

image


please revert this change and publish a new version. For now I am going to fork the repo and remove the `hidden` case and use that in my projects.

**Comment by khoren93** [comment]
@Chappie74, we appreciate your suggestion. Kindly verify it in version v1.4.1.

**Comment by Chappie74** [comment]
1.4.1 fixed the issue. Thanks.

**Pull request merged**